app/uk/gov/hmrc/estates/transformers/amend/AmendPersonalRepTransform.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.estates.transformers.amend
+
+import play.api.libs.json._
+
+trait AmendPersonalRepTransform {
+  def setPersonalRep(input: JsValue, newPersonalRepDetails: JsValue): JsResult[JsValue] = {
+    val personalRepPath = (__ \ 'details \ 'estate \ 'entities \ 'personalRepresentative)
```

Comment:
Can you double check this? Is this the array which contains perRepInd perRepOrg objects?

Reply:
Yes, I've checked and this is the same as Lead Trustee in trusts so I've used the same implementation